IntelliCommand/Services/CommandScopeService.cs

```diff
@@ -175,11 +175,13 @@ private void ParseFile(string filePath)
                     
                     var name = xNameAttribute.Value;
 
-                    if (!this.scopeDefinitions.ContainsKey(id))
-                    {
-                        this.scopeDefinitions.Add(id, name);
-                    }
+//in old version:
+//if some scope's name differs in default VS settings and current user settings, assertion fails.
+//e.g. <Scope Name="XAML Designer" ID="{E9B8485C-1217-4277-9ED6-C825A5AC1968}"/> is such name
+//it is "XAML UI Designer" in default settings files, but Visual Studio Changes it to "XAML Designer" in user's CurentSettings.vssettings
+                    this.scopeDefinitions[id] = name;

```

Comment:
I am happy to take the changes to make it better.

And honestly I have not used VS for the last 3-4 years, which means I have not used this extension as well. So some fresh contributors will be good to have ;)